### PR TITLE
Update to MIT license

### PR DESCRIPTION
**Pull request by erikandre**
Updating license from BSD + Patents to MIT license

### CODE REVIEW

**Comment by erikandre** [comment]
Hi @JoelMarcey I don't see the option to add reviewers so pinging you here instead :)

File: README.md [hunk 1]
```diff
@@ -581,7 +581,7 @@ App Links Measurement Events sends additional information from App Links Intents
  [maven-tasks-link]: https://maven-badges.herokuapp.com/maven-central/com.parse.bolts/bolts-tasks
  [maven-applinks-svg]: https://img.shields.io/maven-central/v/com.parse.bolts/bolts-applinks.svg?label=bolts-applinks&style=flat
  [maven-applinks-link]: https://maven-badges.herokuapp.com/maven-central/com.parse.bolts/bolts-applinks
- [license-svg]: https://img.shields.io/badge/license-BSD-lightgrey.svg?style=flat
+ [license-svg]: https://img.shields.io/badge/license-MIT-lightgrey.svg?style=flat
  [license-link]: https://github.com/BoltsFramework/Bolts-Android/blob/master/LICENSE
 
  [latest]: https://search.maven.org/remote_content?g=com.parse.bolts&a=bolts-tasks&v=LATEST
```
Comment (JoelMarcey) [inline README.md]: Can you add this to the very end of the README.md

```
## License
Bolts-Android is MIT licensed, as found in the LICENSE file.
```

Comment (erikandre) [reply README.md]: Done :)

**Comment by JoelMarcey** [comment]
Not sure if the Travis CI failure is meaningful though.

**Comment by erikandre** [comment]
@JoelMarcey - it seems like the CI build has been failing for the last year :(
I don't really have the bandwidth to fix any actual issue, so I'm going to go ahead and land this.
I also don't have write access to Bolts-Android, so could you mind merging the request?

**Pull request merged**